tools/devops/automation/templates/build/build.yml

```diff
@@ -169,6 +169,13 @@ steps:
     if ($buildReason -eq "PullRequest" -or (($buildReason -eq "Manual" -or $buildReason -eq "IndividualCI") -and ($buildSourceBranchName -eq "merge")) ) {
       $configVars.Add("IsPR", "True")
 
+      if ([string]::IsNullOrEmpty($Env:ghprbPullId)) {
+        $configVars.Add("PrID", $Env:ghprbPullId)
+      } else {
+        $configVars.Add("PrID", "")
+      }
```

Comment:
We still use if for API diff, I'll remove it in a next PR once we have the API diff and Generator which is running atm.

Reply:
is that not identical ?

Reply:
Nope, the yaml is not fwd the env from the previous job. We are using the specific env for the step. No ideal.

Reply:
good catch, I need to rest